misc/python/materialize/version_list.py

```diff
@@ -258,6 +258,7 @@ def get_published_minor_mz_versions(
     newest_first: bool = True,
     limit: int | None = None,
     include_filter: Callable[[MzVersion], bool] | None = None,
+    exclude_current_minor: bool = True,
```

Comment:
Can you give an example what version set we want to achieve? What was `current_version` in https://materializeinc.slack.com/archives/CTESPM7FU/p1710542034279119?thread_ts=1710518433.695219&cid=CTESPM7FU?

Reply:
We want to go v0.91.2 -> v0.92.1, but the test was actually testing v0.92.0 -> v0.92.1

Reply:
I am not sure if we need `exclude_current_minor` or whether `get_published_minor_mz_versions` is not behaving as it should. It should only return one version, the most recent patch version, per minor version. Therefore, its result should not contain both `v0.92.0` and `v0.92.1`...

Reply:
v0.92.1 might not exist yet?

Reply:
Then, I don't understand how we end up comparing `v0.92.0` and `v0.92.1`...
Let me have a further look at the test case.